### ISSUE TITLE
home page of buildah.io makes project look inactive since May 9, 2022

### ISSUE THREAD

**Issue by x448** [containers/buildah.io#105]
The [home page](https://buildah.io/) stopped mentioning new releases after v1.26.1.

Here's a screenshot as of January 31, 2023:

![image](https://user-images.githubusercontent.com/57072051/215785554-e1a4b235-22e1-4c61-a68d-acf3defb3caa.png)

**Comment by rhatdan** [comment]
@TomSweeneyRedHat PTAL

**Comment by TomSweeneyRedHat** [comment]
Yep, trying to catch up.  The latest is in progress for 1.29.0, should be up tomorrow.  For the other releases, I've taken on new work and have not been able to tend to the Release Announcements in the time frames that I would like to.  However, I'm trying to slug aside a few hours a week to get the missing ones completed.
